src/hip/device_util.hip.hh

```diff
@@ -95,22 +141,24 @@ inline float abs(float x)
     return fabsf(x);
 }
 
+//----------------------------------------
 __host__ __device__
 inline double abs(double x)
 {
     return fabs(x);
 }
 
+//----------------------------------------
 __host__ __device__
-inline float abs(hipFloatComplex x)
+inline float abs(rocblas_float_complex x)
 {
-    // CUDA has good implementation,
-    // otherwise use our implementation that scales per LAPACK.
 #ifdef __NVCC__
```

Comment:
Is it normal to have `__NVCC__` since this file is about hip?

Reply:
Yes. The HIP code is generated from the CUDA code. Because they need different implementations, we see both implementations here and in the CUDA device_util.cuh file.